chapter04/c04_family_call.rpy

```diff
@@ -3,7 +3,7 @@ label c04_family_call:
     c "Um... [Pause] H-How... How is your family doing?"
     k "!"
     Kenta's body tensed up at the word "family", as if ice had been thrown down his back.
-    Kenta's hazel eyes narrowed at me, looking somewhat fearful and threatening.
```

Comment:
Let me know if this is something important to the story. Kenta's eyes have previously been described as "emerald" which is very different from hazel. Is it intentional that his eye color changes here?